src/julia-syntax.scm

```diff
@@ -2383,12 +2383,13 @@
 
    'string
    (lambda (e)
+     (define (expand-args s)
+       (if (and (pair? s) (eq? (car s) 'string))
+           (apply append (map expand-args (cdr s)))
+           (list s)))
      (expand-forms
-      `(call (top string) ,@(map (lambda (s)
-                                   (if (and (pair? s) (eq? (car s) 'string))
```

Comment:
I think we should instead change this to `(and (length= s 2) (eq? (car s) 'string) (string? (cadr s)))` to avoid too many changes to which `string` calls happen.